pipeline/alert.go

```diff
@@ -350,6 +351,10 @@ type AlertNode struct {
 	// tick:ignore
 	TalkHandlers []*TalkHandler `tick:"Talk"`
 
+	// Send alert to MQTT
+	// tick:ignore
+	MQTTHandlers []*MQTTHandler `tick:"MQTT"`
```

Comment:
This will read like this in TICKscript:

```
|alert()
    .mQTT()
```

Which is really odd so I think we should change this and the method name below to `Mqtt` so that in TICKscript it is all lower case.